react-api-client/src/runs/useAddCapturePreviewImageToRun.ts

```diff
@@ -0,0 +1,35 @@
+import { useQuery } from 'react-query'
+
+import { addCapturePreviewImageToRun } from '@opentrons/api-client'
+
+import { useHost } from '../api'
+
+import type { AxiosError } from 'axios'
+import type { UseQueryOptions, UseQueryResult } from 'react-query'
+import type {
+  CameraImageSettings,
+  DownloadedImageFileResponse,
+  ErrorResponse,
+} from '@opentrons/api-client'
+
+export function useAddCapturePreviewImageToRun(
```

Comment:
```suggestion
export function useCapturePreviewImageForRun(
```
Nit, I think we should keep the base of the naming convention consistent between the run and global preview image capture hooks.